### DISCUSSION TITLE
If I could find a simple "Donate" link, I'd use it : )

### DISCUSSION THREAD

**Discussion by csss1234** [hifiwi-fi/breadcrum.net#461] [General]
Im a huffduffer refugee, really loving Breadcrum, thank you! 

They only way I see to donate is by becoming a "sponsor" on github, which I've never used before, and for some reason I'm not keen on giving my real name or CC info to github.

**Comment by bcomnes** [comment]
Thank you for the kind words and feedback! One of the goals of mine is to try and set this up as a sustainable indie service by accepting paid subscriptions, but I haven't set that up yet. 

Adding donations is trivial though, I can do that in the meantime. Thank you!

Just to clarify, in what ways would you feel comfortable donating? Paypal links? Crypto? Checks/cash and mail? Anything involving online credit processing requires KYC so it's hard to do those anonymously. 

I do plan to support a free tier indefinitely but I am still working out the economics of what this can be. I hope that its not far off from what I have running today.

**Reply by bcomnes** [reply]
I added a donate page today: https://breadcrum.net/docs/donate/

**Comment by csss1234** [comment]
Paypal link seems broken
￼

**Reply by bcomnes** [reply]
Sorry about that, I had some privacy setting turned on. Should be live now.
